### ISSUE TITLE
App flickering/refreshing rapidly upon load

### ISSUE THREAD

**Issue by GRardB** [Kyon147/laravel-shopify#39]
## Expected Behavior

The app loads gracefully.

## Current Behavior

https://user-images.githubusercontent.com/748180/210988051-f9396d94-85c9-46e0-ab81-9cedc8cfb16c.mov

## Failure Information

There seem to be a bunch of repeat requests made to `https://localhost`, which keep redirecting to `https://localhost/authenticate`. The `/authenticate` calls all return `200` codes, so I'm not sure what's happening.

### Steps to Reproduce

I just need to open the app in the Shopify admin panel.

### Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Package Version: v17.3.4
* Laravel Version: v9.46
* PHP Version: v8.1.12
* Template Engine: Blade (though using React w/ Inertia)
* Using a toolset: Docker (via Laravel Sail)

Perhaps also worth noting, I'm using [`ryoluo/sail-ssl`](https://github.com/ryoluo/sail-ssl) to allow for HTTPS with `localhost`.

Basically, I've set up the app using Laravel Jetstream (Inertia + Vue), [replaced Vue with React](https://github.com/ozziexsh/laravel-jetstream-react), and then set up `laravel-shopify`.

Is this flickering normal for development environments, or might something be going wrong? I couldn't figure out based on the documentation, but I think since I'm using Inertia, I don't need to follow the "SPA" directions in the wiki?

**Comment by Kyon147** [comment]
Hi @GRardB 

I can't this with a brand new project, so it could be down to the set up you are using with the third party packages. 

Have you tried removing the additional packages one by one to see if you no longer see the flicker? I'd probably start with the `sail-ssl` as that would likely cause redirects (but only a guess) and then I'd disable Inertia and see if that changes anything.

**Comment by GRardB** [comment]
Huh, now I'm actually not sure what was causing the issue. I created a new project and didn't run into any issues. After deleting the test project and running `docker system prune`, then booting up the original project again... the problem is gone.

If I run into it again, I'll try to do more digging and follow up so that anyone else who may run into this can have some answers. For now, though, I think we're good.

Thanks!
